llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -23,6 +23,8 @@ static cl::opt<TargetLibraryInfoImpl::AltMathLibrary> ClAltMathLibrary(
     cl::init(TargetLibraryInfoImpl::NoAltMathLibrary),
     cl::values(clEnumValN(TargetLibraryInfoImpl::NoAltMathLibrary, "none",
                           "No alternate math library"),
+               clEnumValN(TargetLibraryInfoImpl::SVMLAltMathLibrary, "altsvml",
+                          "Intel SVML library"),
```

Comment:
would it be better to directly use "svml" like the driver option?

Reply:
@FreddyLeaf You mean replacing "altsvml" by "svml"?

Reply:
yes